java/src/main/java/org/rocksdb/LiveFileMetaData.java

```diff
@@ -9,30 +9,40 @@
  * The full set of metadata associated with each SST file.
  */
 public class LiveFileMetaData extends SstFileMetaData {
-  private final byte[] columnFamilyName;
-  private final int level;
 
   /**
    * Called from JNI C++
    */
-  private LiveFileMetaData(
-      final byte[] columnFamilyName,
-      final int level,
-      final String fileName,
-      final String path,
-      final long size,
-      final long smallestSeqno,
-      final long largestSeqno,
-      final byte[] smallestKey,
-      final byte[] largestKey,
-      final long numReadsSampled,
-      final boolean beingCompacted,
-      final long numEntries,
-      final long numDeletions) {
-    super(fileName, path, size, smallestSeqno, largestSeqno, smallestKey,
-        largestKey, numReadsSampled, beingCompacted, numEntries, numDeletions);
-    this.columnFamilyName = columnFamilyName;
-    this.level = level;
+  private LiveFileMetaData(final long nativeHandle) {
```

Comment:
So do I understand correctly... that your goal is to have calls to LiveFileMetaData return the up-to-date values rather than the values which were cached in the class members previously?